unified-codes/apps/data-service/src/app/api.ts

```diff
@@ -0,0 +1,41 @@
+import fastify from 'fastify';
+import fastifyCors from 'fastify-cors';
+
+import * as Schema from './schema';
+import * as Data from './data';
+import * as Resolvers from './resolvers';
+
+import ApolloService from './ApolloService';
+import { KeyCloakIdentityProvider } from './IdentityProvider';
+
+export const createApolloServer = (_typeDefs, _resolvers, _dataSources, _authenticator) => {
+  const AUTH_URL = 'http://127.0.0.1:9990/auth/realms/unified-codes';
+
+  const typeDefs = _typeDefs ?? Schema.typeDefs;
+  const resolvers = _resolvers ?? Resolvers.resolvers;
+  const dataSources = _dataSources ?? {
+    DgraphDataSource: Data.DgraphDataSource,
+    RxNavDataSource: Data.RxNavDataSource,
+  };
+
+  const identityProviderConfig = { baseUrl: AUTH_URL };
+  const identityProvider = new KeyCloakIdentityProvider(identityProviderConfig);
+
+  const apolloService = new ApolloService(typeDefs, resolvers, dataSources, identityProvider);
+  const apolloServer = apolloService.getServer();
+
+  return apolloServer;
+};
+
+export const createFastifyServer = (apolloServer, _plugins) => {
```

Comment:
IIRC main reason for this is it makes it easier to test, but might need to revist this as couldn't get tests working like they do for the `legacy-api` (might need to directly test the `ApolloServer` instead, which isn't a biggie).